### DISCUSSION TITLE
如何配置qq音乐的cookie

### DISCUSSION THREAD

**Discussion by kailin98** [1715173329/UnblockNeteaseMusic#578] [Q&A]
系统：centos7
配置qq音乐需要的环境变量是直接在/etc/profile中使用export还是其他方式，求告知

**Comment by 1715173329** [comment]
env QQ_COOKIE="xxx" node app.js

**Comment by kailin98** [comment]
![image](https://user-images.githubusercontent.com/40024182/158730764-76f63124-f047-41c9-9a78-0df8351b67de.png)
这样只打开qq音乐源，播放周杰伦的以父之名报错如图

**Reply by pan93412** [reply]
最近 UNM 有更新 QQ 音源。或許是你的 UNM 版本太舊？試試看 rc6 版本。

**Reply by kailin98** [reply]
不会，我刚刚git clone的源码

**Reply by 1715173329** [reply]
请确认您已开通绿钻业务。
